packages/web/src/features/translation/TranslationView.tsx

```diff
@@ -347,7 +352,7 @@ export default function TranslationView() {
                 </p>
               )}
               <ol
-                className={`flex flex-wrap ${
+                className={`flex h-fit grow flex-wrap min-h-0 overflow-auto max-h-full ${
```

Comment:
We need to add `content-start` here to prevent gaps between the rows in the translation line

Reply:
Done